src/Tools/GenerateSdkPackages/make-all.ps1

```diff
@@ -0,0 +1,65 @@
+Param(
+    [string]$version = "26127.00",
+    [string]$branch = "d15rel",
+    [string]$outPath = $null,
+    [string]$fakeSign = $null
+)
+
+set-strictmode -version 2.0
+$ErrorActionPreference="Stop"
+
+try {
+    if ($outPath -eq "") {
```

Comment:
@jasonmalinowski it's not a dumb question.  

There are two ways of forcing default values. 

- Mandatory attribute: if missing the scripts will prompt for values.  So if you forget there is a chance your script ends up hanging Jenkins because its' waiting for info to be typed into a prompt
- Use the `[string]$foo = $(throw ...)` approach.  This is great except when the `throw` executes the script exits with a return value of 0.